src/Microsoft.Performance.SDK.Runtime/Extensibility/FilteredDataRetrieval.cs

```diff
@@ -64,6 +60,34 @@ public object QueryOutput(DataOutputPath dataOutputPath)
             }
         }
 
+        /// <inheritdoc/>
+        public bool TryQueryOutput<T>(DataOutputPath dataOutputPath, out T result)
+        {
+            if (!string.IsNullOrWhiteSpace(dataOutputPath.SourceParserId))
```

Comment:
we should probably get the dataoutput path PR in and then check the DataOutputPathType instead

Reply:
Whichever is checked in first, the other should update this. Either way, I don't care.